index.js

```diff
@@ -17,20 +25,35 @@ module.exports.file = options => {
 			throw new Error('The `name` and `extension` options are mutually exclusive');
 		}
 
-		return path.join(module.exports.directory(), options.name);
+		return path.join(getPath(), options.name);
 	}
 
 	return getPath() + '.' + options.extension.replace(/^\./, '');
 };
 
-module.exports.directory = () => {
+module.exports.directory = getPath;
+
+module.exports.clean = () => {
+	const deletedDirectories = del.sync(cacheDirectories.map(directory => directory + '/**'), {force: true});
```

Comment:
Are the entries in `cacheDirectories` guaranteed not to end in `/`, otherwise `+` will not work.